datastore/db/metadata/inspection/__init__.py

```diff
@@ -93,6 +94,19 @@ class Specifications(BaseModel):
     fr: List[Specification]
 
 
+# Awkwardly named so to avoid name conflict
+class DBInspection(BaseModel):
```

Comment:
We dont need this

Reply:
It's not needed, it's for convenience. I prefer working in the code with models than dictionaries. One advantage: If a data field name changes, it has to be updated everywhere it is used with a dictionary, but only once with a model (vscode's rename feature).

Reply:
I didn't know about that feature, I might need to learn about it to make our tests more dynamic

Reply:
It's akin to renaming a variable name:
![image](https://github.com/user-attachments/assets/a967d35d-698a-44c4-8750-543bed979c6b)